strawberry/aiohttp/test/client.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, Mapping, Optional
+
+from strawberry.test.client import BaseGraphQLTestClient, Response
+
+
+class GraphQLTestClient(BaseGraphQLTestClient):
+    async def query(
```

Comment:
mypy is unhappy because `query` and `request` violate the liskov substitution principle. We could fix this by adjusting the return types in the `BaseGraphQLTestClient` :)

Reply:
I should have fixed it! thanks :)